src/components/billingRecord/IFXBillingRecordList.vue

```diff
@@ -170,6 +169,21 @@ export default {
       }
       return message
     },
+    async getFullBillingRecordByItemIndex(index) {
+      let br = this.items?.[index]
+      if (br?.billingRecordStates?.length) {
+        // Full record is already there
+        return br
+      }
+      if (br.id) {
+        // Go get it
+        br = await this.apiRef.getByID(this.facility.invoicePrefix, br.id)
+        this.items[index] = br
```

Comment:
Does this work? Vue sometimes has issues with reactivity  when updating arrays.

```
Vue cannot detect the following changes to an array:

When you directly set an item with the index, e.g. vm.items[indexOfItem] = newValue
```

You can try this instead if you think it isn't updating properly
```suggestion
        this.$set(this.items, index, br)
```